firebase-firestore/src/main/java/com/google/firebase/firestore/Query.java

```diff
@@ -1224,10 +1224,17 @@ private void validateHasExplicitOrderByForLimitToLast() {
   }
 
   /**
-   * Creates an {@link AggregateQuery} counting the number of documents matching this query.
+   * Returns a query that counts the documents in the result set of this query.
    *
-   * @return An {@link AggregateQuery} object that can be used to count the number of documents in
-   *     the result set of this query.
+   * <p>The returned query, when executed, counts the documents in the result set of this query
+   * <em>without actually downloading the documents</em>. The count is performed on the server and
+   * only the resulting count is downloaded.
```

Comment:
I think this sentence should be removed.. `The count is performed on the server and only the resulting count is downloaded.`

Depending on the SOURCE that is used for `get` call, this may not be true in the future, right?

Reply:
Removed. I was trying to figure out how to word it to convey its efficiency. But I think it's probably fine without this staement.